File: moduloBigNumber.java
import java.math.BigInteger;
public class moduloBigNumber{
	public static void main( String... numbers ){
		try{
			BigInteger dividend = new BigInteger( numbers[ 0 ] );
			BigInteger divisor = new BigInteger( numbers[ 1 ] );
			BigInteger number = dividend.mod( divisor );			
			System.out.print( number.toString( ) );
		}catch( Exception exception ){
			System.err.print( exception.getMessage( ) );
		}
	}
}

File: modulo-big-number.js
/*:
	@include:
		{
			"work": "work"
		}
	@end-include
*/
moduloBigNumber = function moduloBigNumber( dividend, divisor, callback ){
	/*:
		@meta-configuration:
			{
				"numbers": "string|number..."
			}
		@end-meta-configuration
	*/
	work( "javac moduloBigNumber.java && java moduloBigNumber " + numbers.join( " " ), callback );
};